phpBB/adm/style/acp_storage.html

```diff
@@ -0,0 +1,45 @@
+<!-- INCLUDE overall_header.html -->
+
+<a id="maincontent"></a>
+
+<h1>{L_STORAGE_TITLE}</h1>
+
+<p>{L_STORAGE_TITLE_EXPLAIN}</p>
+
+<form id="acp_storage" method="post" action="{U_ACTION}">
+
+	{% for storage in STORAGES %}
+		<fieldset>
+			<legend>{{ lang('STORAGE_' ~ storage.get_name | upper ~ '_TITLE') }}</legend>
+		<dl>
+			<dt><label for="{{ storage.get_name }}">{L_STORAGE_SELECT}{L_COLON}</label><br /><span>{L_STORAGE_SELECT_DESC}</span></dt>
```

Comment:
As we are using twig notation here, please prefer twig notation everywhere :

{L_STORAGE_SELECT} => {{ lang('STORAGE_SELECT') }}